LibSerialize.lua

```diff
@@ -753,23 +739,31 @@ local function CreateSerializer(opts, asyncMode)
     end
     if asyncMode then
         state._async = true
-        for k, v in pairs(defaultAsyncOptions) do
-            state._opts[k] = v
-        end
+        state._yieldCheckFn = opts.yieldCheckFn or defaultYieldCheckFn
     end
+    opts = opts or {}
```

Comment:
Just check the arg against nil instead of wasting memory on an empty table (here and below).